apps/02RockPaperScissors.js

```diff
@@ -10,37 +10,64 @@ var rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  if (hand1 === hand2) {
+    return "It's a tie!"
+  } else if (hand1 === "rock") {
 
+    if (hand2 === "scissors") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+
+  } else if (hand1 === "paper") {
+
+    if (hand2 === "rock") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+
+  } else if (hand1 === "scissors") {
+
+    if (hand2 === "rock") {
+      return "Hand two wins!";
+    } else {
+      return "Hand one wins!";
+    }
+
+  } else {
+    return "Unavailable option selected."
+  }
 }
 
-function getPrompt() {
-  rl.question('hand1: ', (answer1) => {
-    rl.question('hand2: ', (answer2) => {
-      console.log( rockPaperScissors(answer1, answer2) );
-      getPrompt();
+  function getPrompt() {
```

Comment:
I was so confused when we started this, because I thought we were supposed to make the answer2 a computer generated one (a la codecademy), but after reading through this it was apparently much easier that I thought.